src/app/zap-templates/zcl/descriptor-cluster.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Descriptor</name>
+    <code>0xF003</code>
+    <define>DESCRIPTOR_CLUSTER</define>
+    <description>The Descriptor Cluster is meant to replace the support from the Zigbee Device Object (ZDO) for describing a node, its endpoints and clusters.</description>
+    <attribute side="server" code="0x0000" define="DEVICE" type="ARRAY" entryType="DeviceType" length="254" writable="false" optional="false">device</attribute>
```

Comment:
Is there any other way to represent list other than type "ARRAY" to not use fixed length? The spec says the list should have one or more elements, why we set the length to 254 not other number?